.github/workflows/build.yml

```diff
@@ -48,17 +52,28 @@ jobs:
       env:
         CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
       run: bash <(curl -s https://codecov.io/bash) -t $CODECOV_TOKEN  -f coverage.xml
+    - name: Upload binaries to release
+      if: github.event_name == 'release'
+      uses: svenstaro/upload-release-action@v2
+      with:
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
+        file_glob: true
+        file: '*.zip'
+        tag: ${{ github.ref }}
     - uses: actions/setup-ruby@v1
     - name: Send Webhook Notification
```

Comment:
This portion could be removed as GitHub now has their webhooks instead of having double, just a thought?

Reply:
Disabled the hook coming from Github itself since this one has more detail